PyCon material/tutorial_exercises/02-Solve-Subs-Plot.ipynb

```diff
@@ -54,9 +54,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Solve takes two arguments, an equation like $x^2 - 4$ and a variable on which we want to solve, like $x$.  \n",
+    "Solveset takes two arguments, an equation like $x^2 - 4$ and a variable on which we want to solve, like $x$.  \n",
```

Comment:
We should perhaps mention it as `3` instead of two like
`it accepts 2 with 1 optional argument specifying the domain`?